src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,101 @@
+extends:
+- ../System/System
+connector:
+  displayName: Linux - Processes (ps)
+  platforms: Any platform with LinuxOS
+  reliesOn: Linux OsCommands
+  information: Gives process specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - linux
+    criteria:
```

Comment:
Under `detection`, we need to add:

```yaml
disableAutoDetection: true
```

because we don't want this connector to be enabled automatically. Only a manual configuration with the necessary variables will activate the connector on a specified host.